2020BIO101/KBd20200911_notes.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 2020-09-11 Notes
+course: BIO101
+author: @dwlg00
+source: #index
```

Comment:
Please make your file source the Bio Index, and link your file in the bio index as well!